src/types/gravity/artworkResponse.d.ts

```diff
@@ -39,6 +39,7 @@ export interface GravityArtwork {
   inquireable: boolean
   acquireable: boolean
   offerable: boolean
+  offerable_from_inquiry: boolean
```

Comment:
Hey @jo-rs, checking about something here —

I have an open PR (https://github.com/artsy/metaphysics/pull/2964) where I actually delete this `artworkResponse.d.ts` file because as far as I can tell it is unused and no longer in sync with Gravity.

So I wanted to confirm whether we know for a fact this file is in use? Or if this was more of a global search-and-replace kinda thing? Either way my PR will need an update — thanks!

Reply:
I was just following form so: not sure if it's in use 🙁

Reply:
I'm not sure either if it's used. I'm 👍  to remove it if it's already outdated and wait for 😱 , if any.

Reply:
Yeah, seems safe to delete. I'll rebase and do so 👍

Thanks!